packages/components/pie-chip/src/chip.scss

```diff
@@ -1,7 +1,7 @@
 @use '@justeattakeaway/pie-css/scss' as p;
 
 @mixin chip-interactive-states($bg-color, $mode: 'default') {
-    &:hover:not(:disabled) {
+    &:hover:not([disabled]) {
```

Comment:
the css`:disabled` selector is only applied if the element can be natively disabled

Reply:
good spot!